Cargo.toml

```
@@ -15,3 +15,8 @@ tracing-subscriber = "0.2"
 tracing-futures = "0.2"
 structopt = "0.3"
 futures = "0.3"
+inotify = "0.8.3"
+serde = { version = "1.0.104", features = ["derive"] }
+serde_json  = "1.0.27"
+inotify-sys = "0.1.3"
+ libc = "0.2"
```

weird indentation here?
```suggestion
libc = "0.2"
```